app/lib/import_export.rb

```diff
@@ -253,7 +253,8 @@ def excel_export(patients)
         patients.find_in_batches(batch_size: 500) do |patients_group|
           comprehensive_details = comprehensive_details_for_export(patients_group, statuses)
           patients_group.each do |patient|
-            sheet.add_row [patient.id] + comprehensive_details[patient.id].values, { types: Array.new(MONITOREES_LIST_HEADERS.length, :string) }
+            extended_isolation = patient[:extended_isolation]&.strftime('%F') || ''
+            sheet.add_row [patient.id] + comprehensive_details[patient.id].values, { types: Array.new(MONITOREES_LIST_HEADERS.length, :string) } + [extended_isolation]
```

Comment:
did we finalize the decision to only export `extended_isolation` but not have it at all as a blank field for the import? if so, the `extended_isolation` field needs to be added to the first argument in this line and not the second, the second argument also needs to be modified to account for the length of the fields including the additional field that's not already included in `MONITOREES_LIST_HEADERS`, the headers would also need to be updated to include this additional field.

I would probably suggest holding off on adding this field to the comprehensive export for now if we're trying to get this in today because I think this would make the exports/import a bit more complicated and would want to hear @holmesie 's thoughts on it as well

Reply:
Resolved via slack.